src/tools/vfs/vfs_daemon.h

```diff
@@ -25,27 +25,12 @@ enum {
 };
 
 
-#define vfs_error(_fmt, ...) \
-    { \
-        fprintf(stderr, "Error: " _fmt "\n", ##__VA_ARGS__); \
-    }
-
-
-#define vfs_log(_fmt, ...) \
-    { \
-        if (g_opts.verbose) { \
```

Comment:
seems g_opts.verbose is not relevant any more, let's revert this refactoring for now